src/transformers/models/wav2vec2/configuration_wav2vec2.py

```diff
@@ -263,3 +268,10 @@ def __init__(
         # ctc loss
         self.ctc_loss_reduction = ctc_loss_reduction
         self.ctc_zero_infinity = ctc_zero_infinity
+
+        # adaptor
```

Comment:
You write `adaptor` everywhere, but do you mean adapter?

Reply:
Yeah, I also first wrote adapter, but then I noticed that Fairseq uses the name "adaptor" everywhere: https://github.com/pytorch/fairseq/blob/89ec6e7efff867d258947acafc57189b257212d0/fairseq/models/speech_to_text/xm_transformer.py#L199 so I aligned it...not sure if they meant this name or misspelled it as well

Reply:
That's not a real word, so maybe use `adapter`? We don't need to follow blindly every convention of fairseq ;-)
My VSCode is going to light up like a Christmas tree otherwise ^^

Reply:
Sounds good!